tools/flash-shelly.py

```diff
@@ -64,29 +67,21 @@
 logging.addLevelName(logging.TRACE, 'TRACE')
 logging.Logger.trace = functools.partialmethod(logging.Logger.log, logging.TRACE)
 logging.trace = functools.partial(logging.log, logging.TRACE)
-logging.basicConfig(format='%(message)s')
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
+logger.setLevel(logging.TRACE)
+def log_level(level):
```

Comment:
this doesn't have to be a function, make it a map variable.

LOG_LEVEL = {
  '0' : logging.CRITICAL,
  ...
}


...

sh.setLevel(LOG_LEVEL[args.verbose])

Reply:
ah ok

Reply:
done.